components/org.wso2.carbon.identity.api.server.action.management/org.wso2.carbon.identity.api.server.action.management.v1/src/main/java/org/wso2/carbon/identity/api/server/action/management/v1/core/ServerActionManagementService.java

```diff
@@ -365,18 +392,24 @@ private Authentication buildAuthentication(Authentication.Type authType, Map<Str
         }
     }
 
-    private void validateActionType(String actionType) {
+    private Action.ActionTypes validateActionType(String actionType) {
```

Comment:
Shall we rename the class to the resolveActionTypes as ActionTypes is returning in addition to validation. WDYT?